libs/error-stack/src/report.rs

```diff
@@ -726,3 +730,10 @@ impl<Context> Extend<Self> for Report<Context> {
         }
     }
 }
+
+#[cfg(feature = "wasm")]
+impl<Context> From<Report<Context>> for JsValue {
+    fn from(report: Report<Context>) -> Self {
+        JsValue::from_str(&report.to_string())
```

Comment:
This is likely to be a pretty poor experience in most cases for users as the `Display` impl really doesn't encapsulate all of the information within the Report.

If we did go about supporting a `From` impl natively in `error-stack` I think we'd need to think more about what sort of representation we'd want to send